src/main/scala/Query.scala

```diff
@@ -1,86 +0,0 @@
-package com.originate.scalypher
```

Comment:
this file just moved, not sure why github can't figure that out (a few files are showing as deleted that actually were `git mv`ed)